contracts/VetoERC20Voting.sol

```diff
@@ -7,34 +7,57 @@ import "./TransactionHasher.sol";
 import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import "@openzeppelin/contracts/governance/utils/IVotes.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
 
 /// @notice A contract for casting veto votes with an ERC20 votes token
-contract VetoERC20Voting is IVetoERC20Voting, TransactionHasher, Initializable {
-    uint256 public vetoVotesThreshold;
+contract VetoERC20Voting is
+    IVetoERC20Voting,
+    TransactionHasher,
+    Initializable,
+    Ownable
+{
+    uint256 public vetoVotesThreshold; // Number of votes required to veto a transaction
+    uint256 public freezeVotesThreshold; // Number of freeze votes required to activate a freeze
+    uint256 public freezeProposalCreatedBlock; // Block number the freeze proposal was created at
+    uint256 public freezeProposalVoteCount; // Number of accrued freeze votes
+    uint256 public freezeProposalBlockDuration; // Number of blocks a freeze proposal has to succeed
+    uint256 public freezeBlockDuration; // Number of blocks a freeze last, from time of freeze proposal creation
```

Comment:
Adding all this functionality directly in here makes me wonder if we should abstract it away in like `IVetoERC20VotingFreezable` or something similar

Reply:
Agreed! I also agree this could be used as an extension or perhaps a team would like to implement just the freeze feature. @tbwebb22 do you think this would be worth the time?

Reply:
As I am reviewing this @tbwebb22 - I am thinking we should have one contract (VetoGuard) - which inherits the vetoerc20 contract as an extension. This way we can create an interface which is necessary for the VetoGuard to operate and this interface could be used to either have a vetoerc20, erc721, or whatever voting system a usul contract can provide.

This will prob be more versatile and we will not have to deploy multiple contracts.

Reply:
I will not make any changes unless we all agree.

Reply:
Resolving this, as we agreed to abstract away this logic in the future if / when there are multiple contracts using the same functionality from this one.